system-configuration/examples/network_configuration.rs

```diff
@@ -0,0 +1,26 @@
+extern crate system_configuration;
+
```

Comment:
It would be nice if you could make examples start with a short comment on what they are doing/showing. Like we do in the existing ones: https://github.com/mullvad/system-configuration-rs/blob/simplify-bindings/system-configuration/examples/watch_dns.rs#L15